README.rst

```diff
@@ -32,7 +32,7 @@ SPICE is an essential tool for scientists and engineers alike in the planetary
 science field for Solar System Geometry. Please visit the NAIF website for more details about SPICE.
 
 *IMPORTANT*: I have no current affiliation with NASA, NAIF, or JPL. The
-code is provided "as is", use at your own risk.
+code is provided "as is", use at your own risk. However, the NAIF now distributes python "lessons" that use SpiceyPy as the python to spice interface.
```

Comment:
I have already added this to the master branch, but I am not sure why this diff is still being shown by github.